pkg/driver/docker.go

```diff
@@ -86,6 +86,39 @@ func (d *dockerDriver) ImageEnsure(ctx context.Context, image types.ImageOptions
 	return &types.ImageStatusResponse{Status: types.ImagePull, Progress: reader}, nil
 }
 
+func (d *dockerDriver) ImageRemove(ctx context.Context, image string, options types.ImageRemoveOptions) (*types.ImageRemoveResponse, error) {
+	insp, _, err := d.cli.ImageInspectWithRaw(ctx, image)
+	if err != nil {
+		if !errdefs.IsNotFound(err) {
+			return nil, err
+		}
+	}
+
+	if insp.ID == "" {
+		return &types.ImageRemoveResponse{Status: types.ImageRemoved}, nil
+	}
+
+	resp, err := d.cli.ImageRemove(ctx, image, dockertypes.ImageRemoveOptions{
+		Force:         options.Force,
+		PruneChildren: options.PruneChildren,
+	})
+
+	if err != nil {
+		return nil, err
+	}
+
+	deletedItems := make([]types.ImageRemoveResponseItem, 0, len(resp))
```

Comment:
Do you really need to collect this information? You discard it anyway. If non needed, why return?